library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
@nileshprasad137  we also need to get rid of the old version of things when we can instead of 1.9.1 try using 3.1.1